config/rbac/leader_election_role.yaml

```diff
@@ -6,8 +6,10 @@ metadata:
 rules:
 - apiGroups:
   - ""
+  - coordination.k8s.io
   resources:
   - configmaps
+  - leases
```

Comment:
shouldn't these be done here: 👀 👀 
https://github.com/ForgeRock/ds-operator/blob/master/controllers/directoryservice_controller.go#L55-L62